pom.xml

```diff
@@ -194,24 +194,6 @@
     </dependencies>
 
     <profiles>
-        <profile>
```

Comment:
The service descriptor still needs to be excluded from the JAR on Java 9+. But it can probably be moved to the no-spi-on-java9 profile with an exclusion instead of an inclusion if you move it (correctly so, I missed that).

Reply:
Good call.  Excluded in that profile.